BEPs/BEP-563.md

```diff
@@ -0,0 +1,124 @@
+  Description: To make network more efficient between validators.
+</pre>
+
+# BEP-563: Enhanced Validator Network
+- [BEP-563: Enhanced Validator Network](#bep-563-enhanced-validator-network)
+  * [1. Summary](#1-summary)
+  * [2. Status](#2-status)
+  * [3. Motivation](#3-motivation)
+  * [4. Specification](#4-specification)
+    + [4.1 Clarification](#41-clarification)
+      - [4.1.1 Sentry Node](#411-sentry-node)
+      - [4.1.2 NodeID](#412-nodeid)
+    + [4.2 NodeID Registration](#42-nodeid-registration)
+      - [4.2.1 Update on web page](#421-update-on-web-page)
+      - [4,2,2 Update on node start](#422-update-on-node-start)
+    + [4.3 NodeID Verification](#43-nodeid-verification)
+    + [4.4 Message Propagation](#44-message-propagation)
+  * [5. Rationale](#5-rationale)
+    + [5.1 Why add validator’s NodeID](#51-why-add-validators-nodeid)
+    + [5.2 Why use system contract](#52-why-use-system-contract)
+  * [6. Backward Compatibility](#6-backward-compatibility)
+  * [7. License](#7-license)
+
+## 1. Summary
+
+This BEP supports adding validator’s NodeID on the system contract, so the validators can identify each other in the p2p network and forward messages more efficiently.
+
+## 2. Status
+
+Draft
+
+## 3. Motivation
+
+The current BSC P2P network runs well with a 3-second block interval, but after BEP-520 and BEP-524, BSC will achieve a subsecond block interval, which has higher requirements for low message latency.
+
+The current public P2P network is a permissionless network on a large scale, anyone can join the network. Permissionless is necessary for a decentralized network, but it has several challenges, especially on network latency and efficiency.
+
+To satisfy the network requirement of subsecond block interval, it is very helpful to have validators recognize each other in the network and be close enough to each other. Meanwhile, core consensus messages, such as Block Message and Vote Message, can be propagated under the new network topology with better network quality.
+
+## 4. Specification
+
+### 4.1 Clarification
+
+Before diving deep into the design doc, try to introduce some common concepts first.
+
+#### 4.1.1 Sentry Node
+
+For security purposes, most validators work on the intranet. The Sentry Node is a full node that acts as the validator's public network protection node. It connects to the public network P2P Node and forwards P2P messages.
+
+Since the sentry node is the proxy node of the validator on the public network, the validator needs to register the sentry's NodeID on the chain.
+
+#### 4.1.2 NodeID
+
+NodeID is the public key that's used for encrypting communication. A connection can be established with any P2P peer through IP+Port+NodeID.
+
+### 4.2 NodeID Registration
+
+This BEP will upgrade the StakeHub contract; it needs to add new storage slots to save the validator's NodeIDs.
+
+And there are some rules:
+
+- NodeIDs update can only be conducted with operatorAddress or consensusAddress.
```

Comment:
we should allow agent to do this
https://github.com/bnb-chain/bsc-genesis-contract/pull/568